scripts/spack/packages/mfem/fix-dep-ordering.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/makefile b/makefile
+index a4b6296da..b8c62eb21 100644
+--- a/makefile
++++ b/makefile
+@@ -285,7 +285,7 @@ endif
+ 
+ # List of MFEM dependencies, that require the *_LIB variable to be non-empty
+ MFEM_REQ_LIB_DEPS = ENZYME SUPERLU MUMPS METIS FMS CONDUIT SIDRE LAPACK SUNDIALS\
+- SUITESPARSE STRUMPACK GINKGO GNUTLS NETCDF PETSC SLEPC MPFR PUMI HIOP\
++ SUITESPARSE STRUMPACK GINKGO GNUTLS NETCDF SLEPC PETSC MPFR PUMI HIOP\
```

Comment:
Does this need to go into spack or mfem?

Reply:
MFEM. I have a PR for it here https://github.com/mfem/mfem/pull/4363